### PR TITLE
allow symfony 5

### PR DESCRIPTION
**Pull request by Jeroeny**
#SymfonyHackday

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. It seams like there are some updates in the translation component that we need to take care of. Check the PHPStan report.

**Comment by Jeroeny** [comment]
> Thank you. It seams like there are some updates in the translation component that we need to take care of. Check the PHPStan report.

It used a deprecated function indeed. I've added an `if` statement that checks if the function exists. However I'm not sure if an exception should be thrown or - like I've done now - the `$names` should be empty (`[]`).

**Comment by welcoMattic** [comment]
I fix a conflict on composer.json file directly on GitHub. The last thing is to fix cs :+1: 
@Jeroeny could you fix it please?

File: Controller/WebUIController.php [hunk 1]
```diff
@@ -207,9 +207,14 @@ private function getMessageFromRequest(Request $request): Message
...
-        $names = \class_exists(Locales::class)
-            ? Locales::getNames('en')
-            : Intl::getLocaleBundle()->getLocaleNames('en');
+
+        $names = [];
+        if (\class_exists(Locales::class)) {
+            $names = Locales::getNames('en');
+        } elseif (\method_exists(Intl::class, 'getLocaleBundle')) {
+            $names = Intl::getLocaleBundle()->getLocaleNames('en');
+        }
```
Comment (odolbeau) [inline Controller/WebUIController.php]: As this change does not help to make PHPStan happy, could you please revert it?

**Comment by odolbeau** [comment]
I think we can safely ignore PHPStan errors WDYT?
```
 ------ ------------------------------------------------- 
  Line   Controller/WebUIController.php                   
 ------ ------------------------------------------------- 
  215    Call to an undefined static method               
         Symfony\Component\Intl\Intl::getLocaleBundle().  
 ------ ------------------------------------------------- 

 ------ ------------------------------------------------------------------ 
  Line   DependencyInjection/Configuration.php                             
 ------ ------------------------------------------------------------------ 
  39     Call to an undefined method                                       
         Symfony\Component\Config\Definition\Builder\TreeBuilder::root().  
 ------ ------------------------------------------------------------------ 
```

File: Controller/WebUIController.php (hunk 1 above)
Comment (Jeroeny) [reply Controller/WebUIController.php]: Yes, reverted.

**Comment by odolbeau** [comment]
Looks great! WDYT @welcoMattic?

**Comment by welcoMattic** [comment]
LGTM too! Well done @Jeroeny, thank you! 
@odolbeau do you think that we must merge #345 before?

**Comment by odolbeau** [comment]
I tried to rebase #344 on master: this PR is not ready (yet) to be merged.
We should merge this one right now I think.

**Comment by welcoMattic** [comment]
Ok, I let you merge this one :wink:

**Comment by Nyholm** [comment]
Wohoo! Great work!

**Pull request merged**